bcda/cli_test.go

```diff
@@ -223,26 +215,66 @@ func (s *CLITestSuite) TestImportCCLF8_InvalidMetadata() {
 	assert.NotNil(err)
 	assert.EqualError(err, "expected CCLF file number 8, but was 9")
 
-	metadata.cclfNum = 8
-	metadata.filePath = ""
+	metadata = cclfFileMetadata{}
 	err = importCCLF8(metadata)
 	assert.NotNil(err)
-	assert.EqualError(err, "file path must be provided")
+	assert.EqualError(err, "File CCLF8 not found")
 }
 
 func (s *CLITestSuite) TestImportCCLF9() {
 	assert := assert.New(s.T())
+	db := database.GetGORMDbConnection()
+	defer database.Close(db)
 
-	filePath := "../shared_files/cclf/T.A0001.ACO.ZC9Y18.D181120.T1000010"
-	metadata, err := getCCLFFileMetadata(filePath)
-	metadata.filePath = filePath
-	assert.Nil(err)
-	err = importCCLF9(metadata)
+	db.Unscoped().Delete(&models.CCLF9{})
```

Comment:
Maybe also clear out all the `CCLFFile` records so there aren't any hanging out in the table from other tests that could pollute the results of this one.

Reply:
yep :)